examples/longbench/eval_longbench_v2.py

```
@@ -0,0 +1,771 @@
+    )
+
+    outputs = llm.generate(prompts, sampling_params)
+
+    inference_time = time.time() - start_time
+    logger.info(
+        f"Inference completed in {inference_time:.2f} seconds, average time per sample: {inference_time/len(prompts):.3f} seconds"
+    )
+
+    # Process outputs
+    results = []
+    for i, (sample, output) in enumerate(zip(filtered_data, outputs)):
```

Ruff (0.13.1)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)